azure-devops/azext_devops/dev/repos/pull_request.py

```diff
@@ -436,6 +438,32 @@ def add_pull_request_work_items(id, work_items, organization=None, detect=None):
     return wit_client.get_work_items(ids=ids)
 
 
+def checkout(id, remote_name='origin'):  # pylint: disable=redefined-builtin
+    """Checkout the PR source branch locally, if no local changes are present
+    :param id: ID of the pull request.
+    :type id: int
+    :param remote_name: Name of git remote against which PR is raised
+    :type remote_name: str
+    """
+    git_info = get_vsts_info_from_current_remote_url()
+    organization = git_info.uri
+    if not organization:
+        raise CLIError('this command should be used from a valid git repository only')
```

Comment:
nit: **T**his 

Rewording? not just any valid git repository it should be Azure Repo only.